### PR TITLE
Upgrade Aria2D to Latest

### PR DESCRIPTION
**Pull request by uetcis**
The author stopped offering download for previous versions, instead, now he provides a general url. So here I'm updating for this.

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

### CODE REVIEW

File: Casks/aria2d.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 cask 'aria2d' do
-  version '1.2.1,1535339003'
-  sha256 '8338d48fa8c90c8f785568b7b16ca86613b6a8c64d107933a1bc72d6aa401e25'
+  version :latest
```
Comment (vitorgalvao) [inline Casks/aria2d.rb]: The cask has an `appcast`, so `version` and `sha256` should remain (`1.2.4`).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/aria2d.rb (hunk 1 above)
Comment (CaptainYukinoshitaHachiman) [reply Casks/aria2d.rb]: but we don't have the link for different versions

Comment (vitorgalvao) [reply Casks/aria2d.rb]: And set the `sha256`.

Comment (CaptainYukinoshitaHachiman) [reply Casks/aria2d.rb]: But I don't have the version after comma

**Pull request merged**